spec/index.html

```diff
@@ -1243,6 +1250,8 @@ <h2>RDFS Interpretations</h2>
     <em>Ext</em>ension in I) from IC to the set of subsets of IR.</p><p> A class may have an
     empty class extension. Two different classes can have the same class extension.
     The class extension of <code>rdfs:Class</code> contains the class <code>rdfs:Class</code>.</p>
+    
+    <p>RDFS introduces also the special class <code>rdfs:Proposition</code>, i.e. a resource whose extension is the set of propositions as defined in <a href="#simple_entailment_properties">Section 5.3</a>.</p>
```

Comment:
Moving "also", linking the "set of propositions" and making a "smart" section link:
```suggestion
    <p>RDFS also introduces the special class <code>rdfs:Proposition</code>, i.e. a resource whose extension is the <a>set of propositions</a> as defined in [[[#simple_entailment_properties]]].</p>
```
(Note that the automatic section title is then expanded. The part beginning with "as defined in" could be in parentheses; or for that matter even omitted if the also linked "set of propositions" is enough on its own.)